provisioners/configure.sh

```diff
@@ -113,11 +112,16 @@ envsubst \
 a2enmod \
   expires \
   headers \
+  http2 \
+  mpm_event \
   proxy \
+  proxy_fcgi \
   rewrite
+  setenvif \
```

Comment:
(We didn't run into it being used anywhere - is it necessary?)

Reply:
Installing `php-fpm` includes these lines in the apt output:

```
Creating config file /etc/php/7.3/fpm/php.ini with new version
NOTICE: Not enabling PHP 7.3 FPM by default.
NOTICE: To enable PHP 7.3 FPM in Apache2 do:
NOTICE: a2enmod proxy_fcgi setenvif
NOTICE: a2enconf php7.3-fpm
NOTICE: You are seeing this message because you have apache2 package installed.
```

So, we don't use it directly, but it seems to be required.

Reply:
Interesting. Thanks!